extensions/amp-access/0.1/amp-access.js

```diff
@@ -350,6 +352,16 @@ export class AccessService {
             credentials: 'include',
             requireAmpResponseSourceOrigin: true
           }));
+    }).catch(error => {
```

Comment:
I'd rather steer clear of retry initially - that would increase load on target servers.